### PR TITLE
[ENG-761] Log exceptions when running user code

### PR DESCRIPTION
**Pull request by tebeka**
Once logging is merged, top level errors will show in autokitteh.

### CODE REVIEW

File: runtimes/pythonrt/ak_runner.py [hunk 1]
```diff
@@ -236,6 +238,16 @@ def send_response(self, value):
     def extract_response(self, message):
         data = message['payload']['value']
         return pickle.loads(b64decode(data))
+    
+    def send_error(self, error, traceback):
+        message = {
```
Comment (ig-ra) [inline runtimes/pythonrt/ak_runner.py:258]: unrelated to this PR. could we replace message (and payload) by serializable class or similar?

**Review by ig-ra — APPROVED** [review]

File: runtimes/pythonrt/ak_runner.py (hunk 1 above)
Comment (tebeka) [reply runtimes/pythonrt/ak_runner.py:258]: The ideal will be to have a schema for the messages and then generate code both for Go & Python (like protobuf).
But, I don't want to add any external dependency in Python (e.g. protobuf).
Opened ENG-821

Comment (ig-ra) [reply runtimes/pythonrt/ak_runner.py:258]: we could go for less ideal solution, e.g. dataclass or similar

**Comment by tebeka** [comment]
Closing since #230 will send the error to the session log.

**Pull request closed without merging**